### ISSUE TITLE
Status indicator - icons meaning not communicated in text (1.3.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux-theme#133]
### Issue from RE NXT A11y audit 2019
Developers must avoid using CSS Pseudo elements for conveying non-decorative content.

CSS before and after pseudo-elements are used for non-decorative content such as:

- Error icon on the Add actions dialog and Sign In page.

Status indicators communicate meaning (e.g., error, warning) visually without including text for communicating this information. 

### Impact
Users of assistive technology may not have access to the text or image information implemented using CSS Pseudo-elements and may not be interpreted.

### Potential solution

Adding in default text equivalent that is not displayed for each icon (below).
Allow teams to override if default text doesn't make sense for their context.
Allow teams to turn off if redundant with text displayed. Thinking this would mainly be for Success indicator. 

- Error - "Error: "
- Warning -  "Warning: "
- Info - "Important information: "
- Success - "Completed: "

**Comment by Blackbaud-SteveBrush** [comment]
This was fixed on the new status indicator component (see linked PR).
